### PR TITLE
Adding TeamsFeedbackPolicy cmdlets

### PR DESCRIPTION
**Pull request by get-itips**
and improving New- that was already there.

Closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/7047

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsTeamsFeedbackPolicy.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -LocalStore
+Internal Microsoft use
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsTeamsFeedbackPolicy.md:70]:
```suggestion
Internal Microsoft use.
```

File: skype/skype-ps/skype/Get-CsTeamsFeedbackPolicy.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Tenant
+Internal Microsoft use
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsTeamsFeedbackPolicy.md:85]:
```suggestion
Internal Microsoft use.
```

File: skype/skype-ps/skype/Grant-CsTeamsFeedbackPolicy.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DomainController
+Internal Microsoft use
```
Comment (dariomws) [inline skype/skype-ps/skype/Grant-CsTeamsFeedbackPolicy.md:56]:
```suggestion
Internal Microsoft use.
```

File: skype/skype-ps/skype/Grant-CsTeamsFeedbackPolicy.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Tenant
+Internal Microsoft use
```
Comment (dariomws) [inline skype/skype-ps/skype/Grant-CsTeamsFeedbackPolicy.md:131]:
```suggestion
Internal Microsoft use.
```

File: skype/skype-ps/skype/New-CsTeamsFeedbackPolicy.md [hunk 5]
```diff
@@ -99,14 +99,18 @@ Accept wildcard characters: False
 ### -ReceiveSurveysMode
 Set the receiveSurveysMode parameter to enabled to allow users who are assigned the policy to receive the survey.
 
+Possible values:
+ - Enabled
+ - Disabled
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsTeamsFeedbackPolicy.md:104]:
```suggestion
 - Enabled
 - Disabled
 - EnabledUserOverride
```

File: skype/skype-ps/skype/New-CsTeamsFeedbackPolicy.md [hunk 6]
```diff
@@ -99,14 +99,18 @@ Accept wildcard characters: False
...
 ```yaml
-Type: Object
+Type: String
 Parameter Sets: (All)
 Aliases:
 
 Required: False
 Position: Named
-Default value: None
+Default value: Enabled
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsTeamsFeedbackPolicy.md:113]:
```suggestion
Default value: EnabledUserOverride
```

File: skype/skype-ps/skype/Remove-CsTeamsFeedbackPolicy.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Tenant
+Internal Microsoft use
```
Comment (dariomws) [inline skype/skype-ps/skype/Remove-CsTeamsFeedbackPolicy.md:85]:
```suggestion
Internal Microsoft use.
```

File: skype/skype-ps/skype/Set-CsTeamsFeedbackPolicy.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ReceiveSurveysMode
+Set the receiveSurveysMode parameter to enabled to allow users who are assigned the policy to receive the survey.
+
+Possible values:
+ - Enabled
+ - Disabled
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsTeamsFeedbackPolicy.md:105]:
```suggestion
Possible values:
 - Enabled
 - Disabled
```
Same as with New-.

File: skype/skype-ps/skype/Set-CsTeamsFeedbackPolicy.md [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: Enabled
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsTeamsFeedbackPolicy.md:114]:
```suggestion
Default value: None
```

I prefer no default value for Set- but if you disagree, I think it should be the same as New-.

**Review by dariomws — APPROVED** [review]
Approve and also add some suggestions.

**Review by tseward — APPROVED** [review]

File: skype/skype-ps/skype/New-CsTeamsFeedbackPolicy.md (hunk 6 above)
Comment (get-itips) [reply skype/skype-ps/skype/New-CsTeamsFeedbackPolicy.md:114]: image

Did you get EnabledUserOverride in your tenant as default? not on mine...

Comment (dariomws) [reply skype/skype-ps/skype/New-CsTeamsFeedbackPolicy.md:114]: Yes... Awesome.

image

Comment (get-itips) [reply skype/skype-ps/skype/New-CsTeamsFeedbackPolicy.md:114]: Different Tenants... different results... what do you say @tseward should we put "none" as default value then?

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**